src/actions/commands/actions.ts

```diff
@@ -2346,7 +2346,7 @@ class CommandOpenFile extends BaseCommand {
       const fileCommand = new FileCommand({
         name: filePath,
         lineNumber: lineNumber,
-        dontCreateFile: true,
+        createFileIfNotExists: false,
```

Comment:
After changing the negation, the default value is 'false'. Which means you'll need to update the other spot (`:e`) to set this value to `true`.